.github/workflows/nestml-build.yml

```
@@ -0,0 +1,121 @@
+name: NESTML build
+on: push
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+
+    steps:
+      # Checkout the repository contents
+      - name: Checkout NESTML code
+        uses: actions/checkout@v2
+
+      # Setup Python version
+      - name: Setup Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      # Install dependencies
+      - name: Install apt dependencies
+        run: |
+          sudo apt-get install libltdl7-dev libgsl0-dev libncurses5-dev libreadline6-dev pkg-config
+          sudo apt-get install python3-all-dev python3-matplotlib python3-numpy python3-scipy
+
+      # Install Java
+      - name: Install Java 8
+        uses: actions/setup-java@v1
+        with:
+          java-version: '8.0.x'
+          java-package: jre
+
+      # Install Antlr4
+      - name: Install Antlr4
+        run: |
+          wget http://www.antlr.org/download/antlr-4.8-complete.jar
+          echo \#\!/bin/bash > antlr4
+          echo java -cp \"`pwd`/antlr-4.8-complete.jar:$CLASSPATH\" org.antlr.v4.Tool \"\$@\" >> antlr4
+          echo >> antlr4
+          chmod +x antlr4
+          echo PATH=$PATH:`pwd` >> $GITHUB_ENV
+
+      # Install Python dependencies
+      - name: Python dependecies
+        run: |
+          python -m pip install --upgrade pip pytest
+          python -m pip install -r requirements.txt
+          python -m pip install pycodestyle
+
+      - name: Install latest ode-toolbox
+        run: |
+          python -m pip uninstall --yes odetoolbox
+          python -m pip install git+https://github.com/nest/ode-toolbox.git
+
+      - name: Install NESTML
+        run: |
+          echo PYTHONPATH=`pwd` >> $GITHUB_ENV
+          python setup.py install
+
+      - name: Generate Lexer and Parser using Antlr4
+        run: |
+          cd $GITHUB_WORKSPACE
+          find pynestml/generated -not -name __init__.py -a -not -name generated -delete
+          cd pynestml/grammars
+          ./generate_lexer_parser
+
+      # Static code analysis
+      - name: Static code style analysis
+        run: |
+          python3 extras/codeanalysis/check_copyright_headers.py && python3 -m pycodestyle $GITHUB_WORKSPACE -v --ignore=E241,E501,E303,E714,E713,E714,E252,W503 --exclude=$GITHUB_WORKSPACE/doc,$GITHUB_WORKSPACE/.git,$GITHUB_WORKSPACE/NESTML.egg-info,$GITHUB_WORKSPACE/pynestml/generated,$GITHUB_WORKSPACE/extras,$GITHUB_WORKSPACE/build,$GITHUB_WORKSPACE/.github
+
+      # Tests
+      - name: Run unit tests
+        run: |
+          pytest -s -o norecursedirs='*'  -o log_cli=true -o log_cli_level="DEBUG" tests
+
+      # Install NEST simulator
+      - name: NEST simulator
+        run: |
+          python -m pip install cython pytest-xdist
```

```suggestion
          python -m pip install cython
```
I believe this is no longer necessary.